### PR TITLE
8323555: Change JavaFX release version to 22.0.1 in jfx22u

### PR DESCRIPTION
**Pull request by kevinrushforth**
Updates for the beginning of the 22.0.1 release.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] [JDK-8323555](https://bugs.openjdk.org/browse/JDK-8323555) needs maintainer approval

### Issue
 * [JDK-8323555](https://bugs.openjdk.org/browse/JDK-8323555): Change JavaFX release version to 22.0.1 in jfx22u (**Task** - P2 - Approved)

### Reviewers
 * [Johan Vos](https://openjdk.org/census#jvos) (@johanvos - **Reviewer**)
 * [Ambarish Rapte](https://openjdk.org/census#arapte) (@arapte - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx22u.git pull/1/head:pull/1` \
`$ git checkout pull/1`

Update a local copy of the PR: \
`$ git checkout pull/1` \
`$ git pull https://git.openjdk.org/jfx22u.git pull/1/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1`

View PR using the GUI difftool: \
`$ git pr show -t 1`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx22u/pull/1.diff">https://git.openjdk.org/jfx22u/pull/1.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jfx22u/pull/1#issuecomment-1888120153)

### CODE REVIEW

**Comment by kevinrushforth** [comment]
Reviewers: @johanvos or @arapte

**Review by johanvos — APPROVED** [review]

**Review by arapte — APPROVED** [review]

**Comment by kevinrushforth** [comment]
@johanvos Thanks for the review. Can you add the needed maintainer approval, either using the Skara `/approve` command or directly in JBS? No hurry, since I do not plan to announce that this repo is open until next week.

**Pull request closed without merging**
